Resources/Prototypes/Catalog/Fills/Lockers/cargo.yml

```diff
@@ -6,6 +6,8 @@
   - type: StorageFill
     contents:
       - id: ClothingOuterHardsuitSpatio
+      - id: OxygenTankFilled
```

Comment:
Remove these. They can use tank dispensers. If dispensers aren't mapped, we will fix that.